core-bundle/src/Command/DebugContaoTwigCommand.php

```diff
@@ -154,7 +154,7 @@ private function formatMultiline(string $attribute, string $multilineValue, Tabl
         $lines = explode("\n", $multilineValue);
         $rows = [];
 
-        for ($i = 0; $i < \count($lines); ++$i) {
+        for ($i = 0, $c = \count($lines); $i < $c; ++$i) {
             $rows[] = [0 === $i ? $attribute : '', new TableCell($lines[$i], ['style' => $style])];
         }
```

Comment:
```suggestion
        foreach ($lines as $index => $line) {
            $rows[] = [0 === $index ? $attribute : '', new TableCell($line, ['style' => $style])];
        }
```

But not really relevant to this PR, so feel free to ignore.

Reply:
Changed in 8b476f40ab555c8592b59f9e77f2ca84af590f72.

Reply:
And I have also changed the `for` loop in the `SimpleTokenParser` class in 11abda33018da2913f5f02e59fcb23887750f6f0. 😉